logic.js

```diff
@@ -0,0 +1,41 @@
+//Landing-page START
+var groupID;
```

Comment:
can we wrap this in a IFFE to prevent polluting the global scope?

Reply:
lets talk on it later

Reply:
perfect